tests/rt.sh

```diff
@@ -203,9 +217,9 @@ elif [[ $MACHINE_ID = hera.* ]]; then
   ROCOTOCOMPLETE=$(which rocotocomplete)
   ROCOTO_SCHEDULER=slurm
 
-  export PATH=/scratch2/NCEPDEV/fv3-cam/Dusan.Jovic/ecflow/bin:$PATH
-  export PYTHONPATH=/scratch2/NCEPDEV/fv3-cam/Dusan.Jovic/ecflow/lib/python2.7/site-packages
-  ECFLOW_START=/scratch2/NCEPDEV/fv3-cam/Dusan.Jovic/ecflow/bin/ecflow_start.sh
+  export PATH=/scratch2/NCEPDEV/stmp3/Minsuk.Ji/opt/miniconda3/bin:$PATH
```

Comment:
Is this going to be 'permanent' location of miniconda installation? In stmp3?

Reply:
Yes, that is the intention. Do you foresee any issues?

Reply:
Maybe, if they start scrubbing stmp on daily/weekly basis. I do not think software should be installed in /scratch2/NCEPDEV/stmp*

Reply:
I only have access to stmp on both Hera and Orion. Can you cp -r miniconda3 to fv3-cam/Dusan.Jovic on both Hera and Orion?

Reply:
Ok. I see. I can make a copy but I do not think my personal directory is good place for this. Let's keep it as it is until we find more 'permanent' place, f.e. /scratch1/NCEPDEV/nems/emc.nemspara/soft

Reply:
Yes please, let's move away from any code, executables etc in user directories (e.g. Dusan's ecflow installations). I installed ecflow and all other software in shared project space so that it doesn't depend on me or my account.

Have you tried using the GSL miniconda3 module in contrib? It's all there and ready to use, no need to repeat.

Reply:
I am not able to find `ecflow.so` and `ecflow_start.sh` in subdirectories of `/contrib/miniconda3/4.5.12`.

Reply:
Got it - thanks for the info.

Reply:
Dusan/Minsuk, please install the ecflow related code in under emc.nemspara/soft on all the platforms. We can point to them when they are ready (maybe not in this commit if it takes time)

Reply:
Done